cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -46,6 +46,10 @@ public NamespaceConfig(@Nullable String schedulerQueueName, @Nullable String roo
     this.hiveDatabase = hiveDatabase;
   }
 
+  public NamespaceConfig() {
+    this(null, null, null, null);
```

Comment:
does the scheduler queue name support a value of null? (instead of empty string)?